release_docs/RELEASE.txt

```diff
@@ -784,9 +783,6 @@ Platforms Tested
     Centos6                              Version 4.4.7 20120313
     (platypus)                           Version 4.9.3, 5.3.0, 6.2.0
                                      MPICH 3.1.4 compiled with GCC 4.9.3
-                                     PGI C, Fortran, C++ for 64-bit target on
-                                     x86_64;
-                                         Version 19.10-0
 
     Windows 10 x64                  Visual Studio 2015 w/ Intel C/C++/Fortran 18 (cmake)
                                     Visual Studio 2017 w/ Intel C/C++/Fortran 19 (cmake)
```

Comment:
Visual Studio 2019 w/ Intel 19 (C/C++ only - cmake)
should be
Visual Studio 2019 w/ MS C/C++ only - cmake